File: 027.Remove Element/go/027.go
package main

import (
	"fmt"
)

func removeElement(arr []int, val int) int {

	num := 0

	for i, value := range arr {

		if value == val {
			continue
		}

		arr[num] = arr[i]
		num += 1

	}

	return num

}

func main() {

	arr := []int{1, 2, 2, 2, 11, -2, 333, 111, 2, -2, 3333}
	num := removeElement(arr, 2)

	fmt.Println(arr[:num], num)

}

File: 027.Remove Element/python/027.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        if len(nums) == 0:
            return 0
        
        num = 0
        
        for index, value in enumerate(nums):
            if value !=val:
                nums[num]=nums[index]
                num +=1
            
        return num 